zds/tutorialv2/models/database.py

```diff
@@ -179,6 +179,14 @@ def save(self, *args, force_slug_update=False, update_date=True, **kwargs):
         if update_date:
             self.update_date = datetime.now()
         super().save(*args, **kwargs)
+        if hasattr(self, "_authors"):
+            self._authors = list(self.authors.all())
```

Comment:
Je suggère ceci pour que la requête ne soit faite en base de données que si `redacting_authors()` est appelée plus loin dans le code.
```suggestion
            delattr(self, "_authors")
```